drmon.lua

```diff
@@ -32,26 +32,28 @@ local ri
 local action = "None since reboot"
 local emergencyCharge = false
 local emergencyTemp = false
+local finallycharged = 0
+local userStop = -1
 
 monitor = f.periphSearch("monitor")
 inputfluxgate = f.periphSearch("flux_gate")
```

Comment:
For anyone reading this on MineCraft 1.16.5, make sure you're on version 3.0.13.244+ of BrandonsCore and version 3.0.22.444+ of Draconic Evolution. The only changed needed for this to work is changing this line from 
```inputfluxgate = f.periphSearch("flux_gate")``` to ```inputfluxgate = f.periphSearch("flow_gate")```